meta/internal/meta.pb.go

```diff
@@ -1378,15 +1414,11 @@ func (m *JoinRequest) GetAddr() string {
 }
 
 type JoinResponse struct {
-	Header *ResponseHeader `protobuf:"bytes,1,req" json:"Header,omitempty"`
-	// Indicates that this node should take part in the raft cluster.
```

Comment:
Those comments come from here: https://github.com/influxdb/influxdb/blob/master/meta/internal/meta.proto#L316

Looks like the protobuf compiler copies them over when generating this file.  Did you generate this file w/ `go generate`?